mingw-w64-langkit_support/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Fabien Chouteau <chouteau@adacore.com>
+
+_realname=langkit_support
+pkgbase=mingw-w64-${_realname}
+pkgname=${MINGW_PACKAGE_PREFIX}-${_realname}
+pkgver=21.0.0
+pkgrel=1
+
+pkgdesc="Language creation framework (mingw-w64)"
+arch=('any')
+license=('GM GPL3')
+url="https://github.com/AdaCore/langkit"
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc-ada"
+             "${MINGW_PACKAGE_PREFIX}-gprbuild")
+depends=("${MINGW_PACKAGE_PREFIX}-gnatcoll"
+         "${MINGW_PACKAGE_PREFIX}-iconv")
+source=("https://github.com/AdaCore/langkit/releases/download/v${pkgver}/langkit_support-${pkgver}.tar.gz")
+sha256sums=('79609dac4788810b6e5e9be30778476684d8ca08b8bb78f60bab7b3e32f69d27')
+
+build() {
+  cd ${srcdir}/${_realname}-${pkgver}
+  
+  gprbuild -p -P langkit_support.gpr -j$(nproc)
```

Comment:
I suppose the goal of these scripts is to build in production mode: in that case you should add `-XBUILD_MODE=prod`. Also run once per library type (`-XLIBRARY_TYPE=static`, `=static-pic`, `=relocatable`). Same for `gprinstall` commands below.

And same in Libadalang’s `PKGBUILD`

Reply:
Ok, thanks :+1: